src/components/scholarshipList.css

```diff
@@ -0,0 +1,12 @@
+ul{
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+}
+.scholarship-row:after{
+    display: block;
+    height: 6px;
+    background-color: #F5B700;
+    content:"";
+    width: 100%;
+}
```

Comment:
(Feel free to address this in a follow-up PR instead) You could look into [Material UI](https://material-ui.com/)'s components to see if there's something there that helps display the content more cleanly. There's a Data Display section, that includes things like [Lists](https://material-ui.com/components/lists/) and [Tables](https://material-ui.com/components/tables/) which might come in handy.